### ISSUE TITLE
learning rate

### ISSUE THREAD

**Issue by SeeU1119** [wtjiang98/PSGAN#12]
I find that G_lr and D_lr are 2e-5 in code, but in paper is 2e-4, i guess 2e-5 is to adapt finetune process, because i find when i run the code, the model will load the pretrained model, is that right? thanks.

**Comment by wanzysky** [comment]
Both 2e-4 and 2e-5 work well in our tests. The default setting will be changed 2e-4 to be consistent with the paper.
